src/components/tx-flow/flows/AddOwner/ReviewOwner.tsx

```diff
@@ -55,79 +54,44 @@ export const ReviewOwner = ({ params }: { params: AddOwnerFlowProps | ReplaceOwn
-          <Divider />
-          <Box display="flex" flexDirection="column" gap={2} padding={3}>
-            {safe.owners
-              .filter((owner) => !removedOwner || !sameAddress(owner.value, removedOwner.address))
-              .map((owner) => {
-                return (
-                  <EthHashInfo
-                    key={owner.value}
-                    address={owner.value}
-                    shortAddress={false}
-                    showCopyButton
-                    hasExplorer
-                  />
-                )
-              })}
-          </Box>
-          {removedOwner && (
-            <>
-              <div className={css.info}>
-                <Typography variant="overline">Removing owner &darr;</Typography>
-              </div>
-              <Divider />
-              <Box bgcolor="error.light" padding={2}>
-                <EthHashInfo address={removedOwner.address} shortAddress={false} showCopyButton hasExplorer />
-              </Box>
-              <Divider />
-            </>
-          )}
-          <div className={css.info}>
-            <Typography className={css.overline}>Adding new owner &darr;</Typography>
-          </div>
-          <Divider />
-          <Box padding={2}>
-            <EthHashInfo
-              name={newOwner.name}
-              address={newOwner.address}
-              shortAddress={false}
-              showCopyButton
-              hasExplorer
-            />
-          </Box>
-        </Grid>
-      </Grid>
+      <AddressWrapper>
+        <EthHashInfo name={addressBook[safeAddress]} address={safeAddress} shortAddress={false} />
+      </AddressWrapper>
+      <Box display="flex" alignItems="center" gap={2} mx="auto">
+        <SvgIcon component={AddIcon} inheritViewBox fontSize="small" />
+        Add new owner
+      </Box>
+      <AddressWrapper variant="info">
+        <Box sx={{ backgroundColor: ({ palette }) => palette.info.background, borderRadius: '6px', p: '12px' }}>
+          <EthHashInfo
+            name={newOwner.name}
+            address={newOwner.address}
+            shortAddress={false}
+            showCopyButton
+            hasExplorer
+          />
+        </Box>
+      </AddressWrapper>
+      <Divider className={commonCss.nestedDivider} />
+      <Box>
+        <Typography variant="body2">Any transaction requires the confirmation of:</Typography>
+        <Typography>
+          <b>{threshold}</b> out of <b>{safe.owners.length + (removedOwner ? 0 : 1)} owners</b>
+        </Typography>
+      </Box>
+      <Divider className={commonCss.nestedDivider} />
     </SignOrExecuteForm>
   )
 }
+
+// TODO: to be expanded when more cases are added
+const AddressWrapper = ({ variant = 'default', children }: { variant?: 'default' | 'info'; children: ReactNode }) => (
+  <Paper
+    sx={{
+      backgroundColor: ({ palette }) => (variant === 'info' ? palette.info.background : palette.background.main),
+      p: '12px',
+    }}
+  >
```

Comment:
I don't think we need create the `AddressWrapper`. I'd suggest putting the `Paper` components within `ReviewOwner`.